td4m3/src/main/res/layout/generic_card_view.xml

```diff
@@ -20,6 +20,7 @@
 
             <TextView
                 android:id="@+id/cardTitle"
+                android:layout_marginStart="@dimen/card_side_spacing"
```

Comment:
Why do we need this? Don't have this here. This should be done in code, not in the generic component but in your implemented one. We should not put padding here as there may be cases where we don't want padding for some cards, but here you have padding for all cards using generic component.

Reply:
Out of all the comments in this PR, please address this one as highest priority. This is a MUST change. The others can be done later or you can fight against if you think my comment is wrong but this one is a must.